front/src/modules/ui/modal/components/ConfirmationModal.tsx

```diff
@@ -60,26 +64,35 @@ export function ConfirmationModal({
 
   const handleInputConfimrationValueChange = (value: string) => {
     setInputConfirmationValue(value);
-    isValueMatchingUserEmail(confirmationValue, value);
+    isValueMatchingInput(confirmationValue, value);
   };
 
-  const isValueMatchingUserEmail = debounce(
+  const isValueMatchingInput = debounce(
     (value?: string, inputValue?: string) => {
       setIsValidValue(Boolean(value && inputValue && value === inputValue));
     },
     250,
   );
 
+  useScopedHotkeys(
+    [Key.Escape],
+    () => {
+      setIsOpen(false);
+    },
+    ModalHotkeyScope.ConfirmationModal,
```

Comment:
This is actually not needed here because we have a default scope in the modal already